src/main/coordinator.c

```diff
@@ -271,7 +271,7 @@ _finish_req_duration(thr_coord_t* coord)
 	coord->unfinished_threads = rem_threads;
 	// commit this write before signaling the condition variable and releasing
 	// the lock, since it was not atomic
-	as_fence_memory();
+	as_fence_seq();
```

Comment:
Comments to say when we can remove this and possibly why it is here.  Confused on why this fence is needed but don't have the time to investigate :)

Reply:
Same as above